src/mgmt/cli_mgmt.c

```diff
@@ -688,8 +620,7 @@ lookup_sys(const char *name)
 }
 
 static int
-sys_attach(const char *name, int npsrbs, struct psr_buf *psrbs,
-	   struct dc_mgmt_sys **sysp)
+sys_attach(const char *name, struct dc_mgmt_sys **sysp)
```

Comment:
Is the if (name == NULL) check and assignment to DAOS_DEFAULT_SYS_NAME needed here? Since the caller below dc_mgmt_sys_attach() has already done it?

Reply:
You're right; removed.

Reply:
It made sense to me before, that if the name == NULL, then the default name was assigned prior to calling attach() which in turn calls get_attach_info() with the system name.  If I were to have removed anything, it would have been the assignment of the default name that occurs after the attach().  I can study this some more.  Is this behavior what you had in mind?

Reply:
I'm wrong.  I looked again and do not see what I thought I read when I looked at the full source.  I'm sorry for the noise.

Reply:
Ah, OK. No problem at all. Thanks for the quick replies.